README.md

```diff
@@ -23,28 +23,31 @@
+| 3    | How do I motivate myself to start competitive programming                                                    | [https://www.youtube.com/watch?v=GCxwzy9MaY8](https://www.youtube.com/watch?v=GCxwzy9MaY8) | [https://madhavbahl.tech/slides/3](https://madhavbahl.tech/slides/3)           |
+| 4    | How do I get started with data structures, algorithms, problem solving and competitive programming           | [https://www.youtube.com/watch?v=dw2Kdm9YJ-E](https://www.youtube.com/watch?v=dw2Kdm9YJ-E) | [https://madhavbahl.tech/slides/4](https://madhavbahl.tech/slides/4)           |
+| 5    | 8 Simple tips to improve your GitHub profile                                                                 | [https://www.youtube.com/watch?v=4LGF4UhMaSU](https://www.youtube.com/watch?v=4LGF4UhMaSU) | [https://madhavbahl.tech/slides/5/](https://madhavbahl.tech/slides/5/)         |
+| 6    | Which programming language is best for competitive programming and coding interviews \| The Java vs C++ myth | [https://www.youtube.com/watch?v=Ht_VdF5GIBI](https://www.youtube.com/watch?v=Ht_VdF5GIBI) | [http://madhavbahl.tech/slides/6](http://madhavbahl.tech/slides/6)             |
+| 7    | 7 Tips for competitive programmers \| What to do when you aren't able to solve coding questions              | [https://www.youtube.com/watch?v=patIzWIC3OE](https://www.youtube.com/watch?v=patIzWIC3OE) | [http://madhavbahl.tech/slides/7](http://madhavbahl.tech/slides/7)             |
+| 8    | Don't Do These Mistakes As A Software Developer \| 10 Tips for All Software Developers and Students          | [https://www.youtube.com/watch?v=Hj9TziFK0bU](https://www.youtube.com/watch?v=Hj9TziFK0bU) | [https://madhavbahl.tech/slides/8](https://madhavbahl.tech/slides/8)           |
+| 9    | 3 Pieces of advice to computer science undergrads                                                            | [https://www.youtube.com/watch?v=06ZvwAN9BQ0](https://www.youtube.com/watch?v=06ZvwAN9BQ0) | [Coming Soon...](https://www.youtube.com/TheLeanProgrammer?sub_confirmation=1) |
+| 10   | Dont make these 8 mistakes in the early stages of your career 🚫                                             | [https://www.youtube.com/watch?v=skEjScMVXL4](https://www.youtube.com/watch?v=skEjScMVXL4) | [No Slides](https://www.youtube.com/watch?v=skEjScMVXL4)                       |
+| 11   | Dont make this mistake in your life or career 🙅‍♂️                                                             | [https://www.youtube.com/watch?v=qIP-ljy0ShE](https://www.youtube.com/watch?v=qIP-ljy0ShE) | [No Slides](https://www.youtube.com/watch?v=qIP-ljy0ShE)                       |
+
+| 11   | Life of a Computer Engineering student. How to be great coder in college time 🙅‍♂️                                                             | [https://www.youtube.com/watch?v=qIP-ljy0ShE](https://www.youtube.com/watch?v=qIP-ljy0ShE) | [No Slides](https://www.youtube.com/watch?v=qIP-ljy0ShE)                       |
```

Comment:
1. Make it "12"  instead of 11th - 11th video is already there
2. Remove the links to video and slides and mention "To Be Added" in those fields